bundle/compliance/cis_eks/rules/cis_3_2_5/rule.rego

```diff
@@ -9,9 +9,21 @@ rule_evaluation = false {
 	audit.process_contains_key_with_value("--streaming-connection-idle-timeout", "0")
 }
 
+rule_evaluation = false {
+	audit.process_contains_key_with_value("--streaming-connection-idle-timeout", "0s")
+}
+
 # In case both flags and configuration file are specified, the executable argument takes precedence.
 rule_evaluation = false {
 	audit.not_process_arg_comparison("--streaming-connection-idle-timeout", ["streamingConnectionIdleTimeout"], 0)
 }
 
+rule_evaluation = false {
+	audit.not_process_arg_comparison("--streaming-connection-idle-timeout", ["streamingConnectionIdleTimeout"], "0")
+}
+
+rule_evaluation = false {
+	audit.not_process_arg_comparison("--streaming-connection-idle-timeout", ["streamingConnectionIdleTimeout"], "0s")
+}
```

Comment:
external config value can technically be `0`, `"0s"` or `"0"`

Reply:
I think that the `streamingConnectionIdleTimeout` value coming from [kubeconfig](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/) can not be an integer.
Its value is [Duration](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration) which seems to be marshaled to string. ("In particular, it marshals into strings")

Reply:
removed the `int` case